dts/bindings/pwm/st,stm32-pwm.yaml

```diff
@@ -31,4 +31,5 @@ pwm-cells:
   - channel
 # period in terms of nanoseconds
   - period
+# set 'PWM_STM32_COMPLEMENTARY' flag if the complementary output is selected
```

Comment:
note: this comment is not part of the docs, so maybe update the binding description with some examples (it's just `STM32 PWM` now). This way we'll have stuff rendered in docs automatically.